test/functional/detailed_guides_controller_test.rb

```diff
@@ -50,20 +50,6 @@ class DetailedGuidesControllerTest < ActionController::TestCase
     end
   end
 
-  view_test "show includes any links to related mainstream content" do
```

Comment:
But until we actually flick the switch to migrate them, we still need this to work in Whitehall frontend